nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/json/TestWriteJsonResult.java

```diff
@@ -56,6 +59,30 @@
 
 public class TestWriteJsonResult {
 
+    @Test
+    void test() throws IOException {
+        String json = "{\"name\":\"Tom\",\"age\":25,\"address\":[\"Poland\",\"5th avenue\"]}";
+
+        JsonFactory jfactory = new JsonFactory();
+        JsonParser jParser = jfactory.createParser(json);
+
+        Integer parsedAge = null;
+
+        while (jParser.nextToken() != JsonToken.END_OBJECT) {
+            String fieldname = jParser.getCurrentName();
+
+            if ("age".equals(fieldname)) {
+                jParser.nextToken();
+                parsedAge = jParser.getIntValue();
+                break;
+            }
+
+        }
+        jParser.close();
+
+        System.out.println(parsedAge);
```

Comment:
Calls to `System.out.println()` should be avoided in unit tests.
```suggestion
```

Reply:
This test remained there accidentally. I removed it and removed the public modifiers also for consistency with the other test classes in the package.